src/test/java/EulerOneTest.java

```diff
@@ -0,0 +1,72 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EulerOneTest {
+    @Test
+    void whenSetMaxRangeTo10_Give23(){
+        //given
+        int maxRange = 10;
+        //when
```

Comment:
Przed `when` i `then` lepiej dawać spacje - przynajmniej dla mnie jest to bardziej czytelne.